spec/features/peeps_management_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+require_relative 'helpers/session'
+require_relative 'helpers/peeps'
+
+include SessionHelpers
+include PeepsHelpers
+
+feature 'User posts a peep' do
+
+  before(:each) do
+    User.create(email: 'uhoh@danieljohnston.co.uk',
+                handle: 'hrrmm',
+                password: 'testpassword',
+                password_confirmation: 'testpassword')
+  end
+
+  scenario 'and receives confirmation of posting' do
+    sign_in
+    post_peep
+    expect(page).to have_content('Your peep has been posted')
+  end
+
+  scenario 'while not logged in' do
```

Comment:
Your scenario here doesn't describe the behaviour we would expect when a user posts a peep without being logged in.